packages/@aws-cdk/cloudformation-include/lib/short-form-tags.ts

```diff
@@ -0,0 +1,160 @@
+import * as yaml from 'yaml';
+
+const ref = {
+  identify: (value: any) => typeof value === 'string',
+  tag: '!Ref',
+  resolve: (_doc: any, cstNode: any) => {
+    return {
+      Ref: (yaml.parse(cstNode.toString().substring('!Ref'.length))),
+    };
+  },
+};
+const base64 = {
+  identify: (value: any) => typeof value === 'string',
+  tag: '!Base64',
+  resolve: (_doc: any, cstNode: any) => {
+    return {
+      'Fn::Base64': (yaml.parse(cstNode.toString().substring('!Base64'.length))),
+    };
+  },
+};
+const cidr = {
+  identify: (value: any) => typeof value === 'string',
+  tag: '!Cidr',
+  resolve: (_doc: any, cstNode: any) => {
+    return {
+      'Fn::Cidr': (yaml.parse(cstNode.toString().substring('!Cidr'.length))),
+    };
+  },
+};
+const findInMap = {
+  identify: (value: any) => typeof value === 'string',
+  tag: '!FindInMap',
+  resolve: (_doc: any, cstNode: any) => {
+    return {
+      'Fn::FindInMap': (yaml.parse(cstNode.toString().substring('!FindInMap'.length))),
+    };
+  },
+};
+const getAtt = {
+  identify: (value: any) => typeof value === 'string',
+  tag: '!GetAtt',
+  resolve: (_doc: any, cstNode: any) => {
+    const lastDot = cstNode.toString().lastIndexOf('.');
+
+    if (lastDot !== -1) {
+      return {
+        'Fn::GetAtt': [
+          cstNode.toString().substring('!GetAtt '.length, lastDot),
+          yaml.parse((cstNode.toString().substring(lastDot + 1))),
+        ],
+      };
+    }
```

Comment:
This deserves a _lot_ of comments on what's going on here. Looking at this code, I have no idea what's happening :)